### PR TITLE
docs(ee/admin-gui) clarify Admin GUI auth config prerequisite 

### PR DESCRIPTION
**Pull request by UkiahSmith**
The update makes it more specific the steps that need to be taken to get
the Admin GUI protected.

### CODE REVIEW

File: app/enterprise/0.33-x/admin-gui/configuration/authentication.md [hunk 1]
```diff
@@ -72,8 +72,14 @@ HTTP/1.1 201 Created
 
 Save this `"key": "62eb165c070a41d5c1b58d9d3d725ca1"` for use later. Note that this key is separate and distinct from the admin's RBAC token. 
 
-Now you need an RBAC user to associate to this admin. To add the Super Admin, see 
-"[Bootstrapping the first RBAC user - the Super Admin](/enterprise/{{page.kong_version}}/rbac/examples/#bootstrapping-the-first-rbac-user-the-super-admin)".
+When the Admin was created earlier it also created an RBAC user. This user now
```
Comment (coopr) [inline app/enterprise/0.33-x/admin-gui/configuration/authentication.md]: "This user..." refers to "the Admin" or to "an RBAC user"? I suggest you clarify.

File: app/enterprise/0.33-x/admin-gui/configuration/authentication.md [hunk 2]
```diff
@@ -72,8 +72,14 @@ HTTP/1.1 201 Created
...
+needs to have the RBAC Super Admin role added to it. You might need to create
+an RBAC user to associate to this admin. To add the Super Admin, see
```
Comment (coopr) [inline app/enterprise/0.33-x/admin-gui/configuration/authentication.md]: "admin" -> "Admin", I think.

**Comment by coopr** [comment]
LGTM - do we need anyone else to review it @UkiahSmith , or are we ready for merge to master and deploy to production?

**Comment by coopr** [comment]
@UkiahSmith the Type and Scope suggestion is detailed here https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-message-format

**Comment by coopr** [comment]
I imagine the correct "subject line" for the commit would be `docs(ee-admin-gui)`

**Comment by coopr** [comment]
This https://github.com/Kong/docs.konghq.com/pull/831/commits is a good example of well-written Type and Scope for commits.

**Comment by thibaultcha** [comment]
A good practice is to look at the git history of files that were edited and follow the example of previously used commit messages. Dead simple and works for every project.

**Comment by UkiahSmith** [comment]
Updated.

**Comment by coopr** [comment]
@UkiahSmith I see you've updated the title of the PR - nice! I think what was being requested was that you (also?) update the title of the commits https://github.com/Kong/docs.konghq.com/pull/830/commits

**Pull request merged**